adapters/generate_woke_config

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env ruby
```

Comment:
should we rename this file to have a `.rb` suffix?

Reply:
With the shebang we are telling the shell how to execute this script. It's idiomatic in my experience to leave off the extension for standalone scripts/executables like this under the reasoning that the user doesn't need to know what language it is. They simply need to know it's executable (which we've indicated with the file permissions). In this way the implementation of the tool could change (e.g. Ruby to Python) and the user wouldn't need to change any code that executes the script.